thinc/backends/_custom_kernels.py

```diff
@@ -154,7 +155,7 @@ def clipped_linear(
     return out
 
 
-def gelu(X, inplace=False, threshold=6.0, threads_per_block=128, num_blocks=128):
+def gelu(*, X, inplace=False, threshold=6.0, threads_per_block=128, num_blocks=128):
```

Comment:
I think I would personally opt to not make `X` keyword-only, as the name isn't important and it feels intuitive that the first argument should always be the matrix.

Reply:
I first had that as well, but some of the backprop functions also have `X` as an argument, but not as the first argument. But maybe we could make an exception for `X` for the forwards and `dY` for the backwards?

Reply:
> How bwd-compat breaking will this be?

I don't think this should break anything, since we only use these in `CupyOps` and it's a private module.

Reply:
To me it feels a bit weird to have "required" keyword arguments, i.e. keyword-arguments that don't have a default value. You'd need to know exactly what the name is, like `seq=...` for `seq2col`, where really the name feels arbitrary to me...

Reply:
I have updated the PR to make only non-required arguments kwarg-only.